lib/cortex/snippets/client/helper.rb

```diff
@@ -1,9 +1,11 @@
+require 'cortex/snippets/client/webpage'
+
 module Cortex
-  module Snippets
+  module Snippet
     module Client
       module Helper
-        def snippet(options = {}, &block)
-          snippets = webpage[:snippets] || []
+        def snippet(cortex_client, options = {}, &block)
```

Comment:
I still think we'll want to wrap around this helper in employer - having to pass in the relevant `cortex_client` across our codebase, in our views, might not be ideal